### DISCUSSION TITLE
Bottom navigation sneak preview posted on Facebook

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#12125] [Show and tell]
Just to get some inspiration, I posted a mock of the new home screen to our channel on Facebook.

Cant find a permalink though.

Will post relevant constructive replies here.

Edit:
Not sure if visible without login:
https://www.facebook.com/android.geocaching/posts/10159232987175091

**Comment by moving-bits** [comment]
> Edit: Not sure if visible without login:

sadly, no

But anyway, thanks for posting the article!

**Reply by Lineflyer** [reply]
Strange...the link worked for me in an incognito tab of Chrome

**Reply by moving-bits** [reply]
I do not have a FB account, and following the link only lead to a login screen for me

**Reply by fm-sys** [reply]
For me, it seems to work on mobile browsers without login, but not on a desktop PCs...

**Reply by moving-bits** [reply]
Nice hint - for me the mobile version does work as well (after closing some cookie selection banner etc.)...

**Comment by fm-sys** [comment]
Not sure how much constructive feedback we will receive via this way. It's difficult to judge about something you never have used by yourself...

Anyway, there seem to be three user groups:
- "why do you change things which had worked?"
- users who haven't yet understand the concept / cannot imagine how it will work ("how many clicks will be required to access e.g. offline lists? Things should stay directly accessible")
- users who have understood the basic concept and are looking forward to be able to easily switch from everywhere inside the app

So I guess quite some users doesn't have understand how things will really work. While other problems we had discussed like "less space on the map" where not mentioned at all.

Also, we should not forget that this is by far the most prominent UI change in the last months of development. While other changes did only "look" slightly different but behave still the same, the bottom nav will indeed completely change how cgeo will behave. People are creatures of habit. They will need their time to get used to it...

**Reply by ztNFny** [reply]
I don't think FB is the best place for such discussions.

Is there the possibility in Play Store to have different Beta groups? Like could we do an alpha group with limited size?

**Reply by moving-bits** [reply]
IIRC we can do both alpha releases to dedicated testers as well as staged beta releases (releases targeting only a certain percentage, a specific country etc.).

**Reply by eddiemuc** [reply]
I always thought of the nightly users to be our "alpha tester group".

**Reply by Lineflyer** [reply]
Yes, nightlies are kind of our alpha builds. We disregarded to publish them on Play in some close test channels because of the effort. IMHO its OK this way. Nightlies are publicly available.

**Comment by eddiemuc** [comment]
Looking at FB comments I have to agree with @fm-sys that seemingly many users haven't understand the concept from the single screenshot. They seem to think that just the home screen has changed and all other screens stay the same (this may also be the reason why there are no complains about "wasted map space").

I would suggest to post mockups from at least three of the pages (e.g. home, map, stored-lists - all with bottom navigation) to clear up this misunderstanding.

**Reply by Lineflyer** [reply]
The FB posting was just meant to give a short preview....once its in beta phase we can do such slideshow/video. For me the posting was just to maybe read some comment with a topic we haven't thought about yet.

Besides this: As always in such occasions most commenters are negative (and the silent majority maybe leaves a like only). So we should not treat the amount of negative feedback as a problem by now.

**Comment by ztNFny** [comment]
> I would suggest to post mockups from at least three of the pages 

Why not a video? Shows a lot more than just stills
